crypto/bio/b_sock2.c

```diff
@@ -206,7 +206,10 @@ int BIO_listen(int sock, const BIO_ADDR *addr, int options)
     }
 
 # ifdef IPV6_V6ONLY
-    if ((options & BIO_SOCK_V6_ONLY) && BIO_ADDR_family(addr) == AF_INET6) {
+    if (BIO_ADDR_family(addr) == AF_INET6) {
+        /* Note: Windows default of IPV6_V6ONLY is ON, and Linux is OFF.
+         * Therefore we always have to use setsockopt here. */
```

Comment:
Block comment must be like this:

``` C
/*
 * text...
 * text...
 */
```

Reply:
Ah, okay, I did just copy the style from a few lines above.
Will fix.

Reply:
maybe I fix the other comment as well.